src/lib.rs

```diff
@@ -92,6 +92,27 @@
 //! }
 //! ```
 //!
+//! # Available logging frameworks
+//!
+//! In order to produce log output executables have to utilize
+//! a framework compatible with the facade.
```

Comment:
Thanks, I didn't notice how pervasive it is in the existing documentation. I filed #191 to follow up.